stylesheets/styles.css

```diff
@@ -886,6 +886,11 @@ div.wiki-header button.close-button, div.wiki-header button.open-button {
   margin-top: -120px;
 }
 
+#branchesTab .list-group-item, #tagsTab .list-group-item{
+  width:300px;
+  margin: px;
+} 
+
```

Comment:
These styles also seem like they could be in a header.component.css file instead of a global stylesheet